### ISSUE TITLE
compilation failed - wasm-ld doesn't exist

### ISSUE THREAD

**Issue by ghost** [maniyar1/wasm-canvas-demo#3]
Hmm... It seems to be trying to use a wasm linker that doesn't exist. Any thoughts?

`make` output:
```
clang --target=wasm32 -nostdlib -o web/bin/app.wasm c/*.c
clang: error: unable to execute command: Executable "wasm-ld" doesn't exist!
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

**Comment by maniyar1** [comment]
Compiles for me, wasm-ld is part of the lld package on Arch it seems. Might need to find what package it is for your distro/system?

**Comment by ghost** [comment]
I've moved on to other projects, but I'll be sure to try... Maybe a simple `sudo apt-get install lld` or `sudo apt-get install wasm-ld`?

**Comment by maniyar1** [comment]
Yeah, thanks for the report regardless. I'll keep this issue open in case someone else runs into problems.
